service-python/Dockerfile.local

```diff
@@ -1,16 +1,16 @@
-ARG PYTHON_VERSION=3.9
+ARG WKHTML_VERSION=3.10.6-0.12.6-small
 
-FROM python:${PYTHON_VERSION}-alpine
+FROM surnet/alpine-python-wkhtmltopdf:${WKHTML_VERSION}
 LABEL org.opencontainers.image.source=https://github.com/department-of-veterans-affairs/abd-vro
 
+ARG SERVICE_FOLDER=pdfgenerator
```

Comment:
Wondering if the default value should be some value that will generate an error and fail if `SERVICE_FOLDER` is not set.
Along the same lines, can we add something to cause the docker build to fail if `SERVICE_FOLDER` is set incorrectly?

Reply:
The `SERVICE_FOLDER` defaults to `gradle` which will select the Gradle build instructions if the argument is not provided. If someone does provide an value for that argument then it will attempt to select the folder in which case a valid argument will build properly while a bad one will automatically fail